packages/manager/modules/vrack/src/move-dialog/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I know the texts were given by the PU but there are way too many mistakes (not your fault of course 😄 )

Reply:
I just moved the translations from another file to this place. Didn't know that there would be so many corrections to this :D

Reply:
I've seen that, thanks for the hard work 👍